Neos.Cache/Classes/Backend/TaggableBackendInterface.php

```diff
@@ -29,6 +29,15 @@ interface TaggableBackendInterface extends BackendInterface
      */
     public function flushByTag(string $tag): int;
 
+    /**
+     * Removes all cache entries of this cache which are tagged by any of the specified tags.
+     *
+     * @param array<string> $tags The tags the entries must have
+     * @return integer The number of entries which have been affected by this flush or NULL if the number is unknown
+     * @api
+     */
+    public function flushByTags(array $tags): int;
```

Comment:
thx, good point, then this is incorrect since 2017 when the type hint for `flushByTag` was added. I copied the description without noticing the mention of the NULL value. Therefore I would adjust the docs for both methods.